qiskit/algorithms/time_evolvers/pvqd/pvqd.py

```diff
@@ -346,8 +356,13 @@ def evolve(self, evolution_problem: EvolutionProblem) -> EvolutionResult:
 
         # get the function to evaluate the observables for a given set of ansatz parameters
         if observables is not None:
+            if self.estimator is None:
+                warnings.warn(
+                    "The evolution problem contained aux_operators but no estimator was provided. "
+                    "The algorithm continues without calculating these quantities. "
```

Comment:
Similar comment that I made elsewhere. I think perhaps this is more a user config error where it would be better to raise an error rather than continuing. Have them fix the situation...

Reply:
And I will add a relevant unit test.